test/units/testsuite-70.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -ex
+set -o pipefail
+
+export SYSTEMD_LOG_LEVEL=debug
+
+# Prepare fresh disk image
+dd if=/dev/zero of=/test.img bs=1024k count=100 status=none
```

Comment:
put this on /tmp, / might not be writable

Reply:
And that results in an OOM error. :) But making the image smaller works.

Reply:
Maybe using `/var/tmp` instead of `/tmp` (which is tmpfs) should work better/more reliably.